api/src/main/kotlin/io/spine/protodata/event/EnumCompilerEvents.kt

```diff
@@ -68,13 +68,15 @@ internal class EnumCompilerEvents(
             EnumEntered.newBuilder()
```

Comment:
Please consider using Kotlin DSL for Protobuf. It should be already available here. I've just tried it in another branch and it worked for me.